### ISSUE TITLE
All good but can't access it

### ISSUE THREAD

**Issue by emircanerkul** [eclipse/che#22989]
### Describe the bug

I created minicube in my proxmox and setup che well. All looks good it gave me https://192.168.49.2.nip.io address which is non-accessible ip. My machine ip is 192.168.1.200 but docker utilizes that network and I can't access it. 

Isn't this issue known? How can i connect my dashboard?

https://github.com/eclipse-che/che/issues/21332 document is not clear.

### Che version

7.86@latest

### Steps to reproduce

- Create VM in proxmox
- Follow https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-minikube/

### Expected behavior

Able to access it.

### Runtime

minikube

### Installation method

chectl/latest

### Environment

Linux

**Comment by emircanerkul** [comment]
Hi @akurinnoy 

Are there any progress?

I was trying to install che on wsl with minikube, but  facing same issue, i can not access.

So far i able to open dashboard but looks i missed something. Sorry for being k8s noob. But docs are really not complete.

![image](https://github.com/eclipse-che/che/assets/11430795/b57e621c-6561-4881-8815-aec175367f7d)

@RomanNikitenko if you can please could you prioritize this problem in the backlog? I think this really easy for someone has knowledge of k8s. est. 15-30m

Thankx

**Comment by RomanNikitenko** [comment]
@tolusha 
could you take a look please

**Comment by tolusha** [comment]
Hello. @emircanerkul
For our cases, we deploy minikube on local machine but not in some Virtual Environment.

**Comment by emircanerkul** [comment]
@tolusha I see, due to i'm using windows i thought using wsl would be good option. Could you check my second usecase https://github.com/eclipse-che/che/issues/22989#issuecomment-2180820370

thank you

**Comment by emircanerkul** [comment]
@tolusha are u sure about https://github.com/eclipse-che/che/issues/22989#issuecomment-2182858270? I also tried without wsl (but anyway docker desktop uses wsl backend) it gave same problem. Minikube has own sub network (https://192.168.49.2.nip.io/dashboard) and i can not access it. My ip/subnet is 192.168.1.xx

**Comment by huonguyenlt** [comment]
have you been able to fix the issue.? I have the same issue

**Comment by emircanerkul** [comment]
@huonguyenlt no, i gave up. After thinking i decided that i do not needed :) actually. Also explored https://coolify.io and currently doing some custom setup regarding remote development.
